package.json

```diff
@@ -7,7 +7,7 @@
     "./picocolors.js": "./picocolors.browser.js"
   },
   "sideEffects": false,
-  "description": "The tiniest and the fastest coloring library ever",
+  "description": "14s smaller than chalk and 2x faster",
```

Comment:
This will appear in npm search and may look confusing as people may not be familiar with chalk
![image](https://user-images.githubusercontent.com/5635476/136019101-04cdce02-a78d-4ad5-be5d-dc69e2f1bc93.png)

Reply:
Yeap, we can revert it back

Reply:
Fixed